app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,25 @@
+@import "base/fonts";
```

Comment:
do we need to import this, if it's already loaded? (base is before components) @emilebosch ?